src/engine/test/source/base/parseEvent_test.cpp

```diff
@@ -13,129 +13,69 @@ constexpr char TEST_IPV4[] {"1.205.0.44"};
 constexpr char TEST_EXTENDED_IPV6[] {"ABCD:EFG0:1234:5678:0009:00AB:00C6:0D7B"};
```

Comment:
We should add test for the escpae | char once implemented

Reply:
implemented